### PR TITLE
Add new folder and files for security-area tutorial

### PR DESCRIPTION
**Pull request by shannonxtreme**
Create directory structure for GKE security focused tutorials.

Add sample files for GKE tutorial to use Workload Identity with Secret Manager.

### CODE REVIEW

**Comment by NimJay** [comment]
Hi @shannonxtreme! :)

1. Could you please provide more context for these changes?
2. Do you have a link to the draft of the associated tutorial (e.g., a Google Doc or a CL)?

Thank you.

**Comment by shannonxtreme** [comment]
@bourgeoisor I added commits to build and push the app to the google-samples repo. LMK if the syntax etc is OK

**Comment by bourgeoisor** [comment]
To keep consistency, could you move the files in `src/` one directory up, and delete the `src/` directory?

Example / precedent: https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/tree/main/hello-app

File: security/wi-secrets/manifests/bq-readonly-key [hunk 1]
```diff
@@ -0,0 +1 @@
+key=my-api-key
```
Comment (bourgeoisor) [inline security/wi-secrets/manifests/bq-readonly-key:1]: Is this supposed to be a YAML file maybe? If so maybe let's add a .yaml suffix to the file name.

Comment (bourgeoisor) [reply security/wi-secrets/manifests/bq-readonly-key:1]: Could we also have a comment above the key stating e.g.:
```
# It is not recommended practices to check in secrets into a source code repository
```

File: security/wi-secrets/main.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
+package main
```
Comment (bourgeoisor) [inline security/wi-secrets/main.go:1]: Missing Google copyrights header

File: security/wi-secrets/manifests/admin-pod.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```
Comment (bourgeoisor) [inline security/wi-secrets/manifests/admin-pod.yaml:1]: Missing Google copyrights header

File: security/wi-secrets/manifests/readonly-pod.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```
Comment (bourgeoisor) [inline security/wi-secrets/manifests/readonly-pod.yaml:1]: Missing Google copyrights header

File: security/wi-secrets/manifests/secret-app.yaml [hunk 5]
```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
```
Comment (bourgeoisor) [inline security/wi-secrets/manifests/secret-app.yaml:1]: Missing Google copyrights header

File: security/wi-secrets/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,13 @@
+FROM golang:1.18 as builder
```
Comment (bourgeoisor) [inline security/wi-secrets/Dockerfile:1]: Missing Google copyrights header

File: security/wi-secrets/manifests/bq-readonly-key (hunk 1 above)
Comment (shannonxtreme) [reply security/wi-secrets/manifests/bq-readonly-key:1]: No it's meant to just be a file. Do you know if the `--data-file` flag in kubectl will accidentally parse the comment as well though ?

Comment (bourgeoisor) [reply security/wi-secrets/manifests/bq-readonly-key:1]: Nah you're right, it's meant to just be a file, let's leave as-is :)

**Review by bourgeoisor — APPROVED** [review]
This generally looks good -- we can do further tweaks at the same time as the doc / tutorial gets reviewed if need be. We can put this in the repo to trigger the image build and that'll help with testing.

**Comment by shannonxtreme** [comment]
(rebased main)

**Pull request merged**